vendor/github.com/tudalex/seccomp-sandbox/sandbox.go

```diff
@@ -0,0 +1,98 @@
+package sandbox
+
+/*
```

Comment:
Yeah we'll need to figure out what we want to do for the other platforms here. We might need to have stub functions that do nothing for Windows and OSX so the build does not fail on these platforms when it's linked with this library. I imagine we could do that here, or if we want to keep this library seccomp focused, we could wrap the jail call in another function in mig (e.g., sandbox_linux.go would contain a function that calls this, and imports the library), _windows _osx would just be a noop